utils/S3UtilsMongoClient.js

```diff
@@ -838,62 +838,6 @@ class S3UtilsMongoClient extends MongoClientInterface {
         }
     }
 
-    /*
```

Comment:
I don't remember why we had a re-implementation of this function, were we doing more logic here? Just making sure we are not deleting something we should not

Reply:
it looks like what is described:

> Overwrite the getBucketInfos method to specially handle the cases that bucket collection exists but bucket is not in metastore collection.
> For now, to make the count-items cronjob more robust, we ignore those "bad buckets"

But this is now the behavior in arsenal